src/load_games.py

```diff
@@ -5,14 +5,57 @@
     from src.load_games import get_games
     games = get_games('resources/PGN_database')
 """
-import ast
 import os
 import warnings
 from operator import itemgetter
+import collections
 
+import chess.pgn
 
-def load_games(path: str = 'resources/PGN_database',
-               is_rated: bool = None):
+
+def flatten_list(arg_list):
+    """Flattens a list completely.
+
+    Parameters
+    ----------
+        arg_list : list[list[...]]
+            nested list
+
+    Returns
+    -------
+        list[...]
+            flattened list of games in list.
+
+    """
+    return [item for sublist in arg_list
```

Comment:
It should be clear in the documentation this function only transforms 2-level lists in 1-level lists.